### PR TITLE
docs/v1 folder for SG 1.0.0

### PR DESCRIPTION
**Pull request by golkir**
A new docs/v1 folder for SG v1 incorporates UI examples, GIFs, and screenshots and makes updates to Load Balancer, Kube, Cloud Accounts and other resources.

### CODE REVIEW

File: docs/v1/cloud_account.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Cloud Account
+
+A Cloud Account is a simple object that holds API access credentials (e.g., API Token, security keys) for any supported Provider (e.g., GCE, AWS, Packet). It is the parent object of [Kubes](kube.md).  Credentials are validated by an API call to the respective Provider.
+
+### DigitalOcean
+
+![](https://s3-ap-southeast-2.amazonaws.com/sg-github-wiki-images/linking-cloud-accs/digital-ocean-credentials.png)
```
Comment (saheienko) [inline docs/v1/cloud_account.md]: Hi, @golkir. Thanks for your contribution.
Can you place this assets (.png and .gif files) under the img/images directory and use [relative links](https://help.github.com/articles/about-readmes/#relative-links-and-image-paths-in-readme-files) for them?

Comment (golkir) [reply docs/v1/cloud_account.md]: Hi, @saheienko. Ok, I'll do this. Thanks.

**Comment by TheKLARKEN** [comment]
Looks good! 🍰

**Comment by TheKLARKEN** [comment]
@gopherstein I don't think our approvals caused a merge; could you do one? Thanks!!

**Pull request merged**